ractor/src/actor/actor_properties.rs

```diff
@@ -47,6 +58,7 @@ pub(crate) struct ActorProperties {
     pub(crate) type_id: std::any::TypeId,
     #[cfg(feature = "cluster")]
     pub(crate) supports_remoting: bool,
+    pub(crate) member_ship: Mutex<Option<MemberShip>>,
```

Comment:
nit: `membership`

Reply:
The biggest concern however, why does it require that the PG membership be a property of the actor cell directly? Why can it not be isolated, as it is today?

How does this solve the potential race condition in PG?